programs/bash-completion/completions/clickhouse-bootstrap

```diff
@@ -0,0 +1,81 @@
+#
+# bash autocomplete, that can work with:
+# a) --help of program
+#
+# Also you may like:
+# $ bind "set completion-ignore-case on"
+# $ bind "set show-all-if-ambiguous on"
+#
+# It uses bash-completion dynamic loader.
+
+# Known to work with bash 3.* with programmable completion and extended
+# pattern matching enabled (use 'shopt -s extglob progcomp' to enable
+# these if they are not already enabled).
+shopt -s extglob
+
+export _CLICKHOUSE_COMPLETION_LOADED=1
+
+function _clickhouse_bin_exist()
+{ [ -x "$1" ] || command -v "$1" >& /dev/null; }
+
+function _clickhouse_quote()
+{
+    local quoted=${1//\'/\'\\\'\'};
+    printf "'%s'" "$quoted"
+}
+
+# Extract every option (everything that starts with "-") from the --help dialog.
+function _clickhouse_get_options()
+{
+    "$@" --help 2>&1 | awk '{ for (i=0; i < NF; ++i) { if (substr($i, 0, 1) == "-" && length($i) > 1) print $i; } }' | sort -u
```

Comment:
It should give only dashes and option name, right? And remove `=`, `,`, `<....>` and so on. Looks this variant doesn't work:
```
akuzm@akuzm-nix:~/ch1$ $BUILD/programs/clickhouse-server --help | awk '{ for (i=0; i < NF; ++i) { if (substr($i, 0, 1) == "-" && length($i) > 1) print $i; } }' | sort -u
-C<file>,
-C<file>, --config-file=<file>    load configuration from a given file
--config-file=<file>
--daemon
...
```

Reply:
I.e. should it be like this?
```
-C
--config-file
--daemon
....
```

Reply:
You are right, forget about these these server specific non-boost options

Reply:
Fixed, actually right now there is no completion for `clickhouse-server` (but there is completion for `clickhouse server ...`), since it does not have lot's of options (well actually it allows override anything from the config, but there is no way to get all these keys right now, so it is not implemented)